components/lsp/SwitchWalletLsp.styled.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import { BaseButton } from "..";
+
+export const Disclaimer = styled.p`
+  margin: 0 0 26px;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 22px;
```

Comment:
2 nits, f
- font-size is set in px, which would not allow a user to rescale it if they changed the browser default font-size, a nice trick to set the font in px, but allowing the user to rescale is doing `font-size: ${20 / 16}rem`, which will be the rem value of 20px at 1rem = 16px, but will scale
- Setting line-height in px won't rescale it, I suggest we set it in absolute value like `line-height: ${22 / 20}` if we are targeting 22px of line-height at 1 rem = 16px